phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexAsyncThresholdIT.java

```diff
@@ -162,9 +162,7 @@ public void testAsyncIndexCreation() throws Exception {
                 exception = (SQLException) e;
             }
             connection.commit();
-            PTableKey key = new PTableKey(null, this.tableName);
-            PMetaData metaCache = connection.unwrap(PhoenixConnection.class).getMetaDataCache();
-            List<PTable> indexes = metaCache.getTableRef(key).getTable().getIndexes();
+            List<PTable> indexes = connection.unwrap(PhoenixConnection.class).getTable(this.tableName).getIndexes();
```

Comment:
why this change?

Reply:
This is to make sure we fetch the latest ptable. I think such changes are okay to keep even if UCF=always.